pkg/store/proxy.go

```diff
@@ -68,6 +68,7 @@ type Client interface {
 // ProxyStore implements the store API that proxies request to all given underlying stores.
 type ProxyStore struct {
 	logger         log.Logger
+	debug          bool
```

Comment:
We can iterate on this but would be nice to have a builder pattern, with option methods like `WithProxyDebugLogging` similar to `BucketStore`. That way we can avoid having to change signature everytime. 🙂